File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.swift
import Foundation

let word = readLine()!.map{String($0)}
var alpabet_cnt = Array(repeating: 0, count: 26)

for w in word {
    let a = Int(UnicodeScalar(w)!.value)
    alpabet_cnt[a-97] += 1
}

var answer = ""
for c in alpabet_cnt { answer += "\(c) " }
print(answer)

File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        char[] word = br.readLine().toCharArray();
        int[] a_cnt = new int[26];

        for(int i=0; i<word.length; i++) a_cnt[(int)word[i] - 97] += 1;

        String answer = "";
        for(int i=0; i<a_cnt.length; i++) answer += a_cnt[i] + " ";
        System.out.println(answer);
    }
}